driver-mapping/src/main/java/com/datastax/driver/mapping/QueryType.java

```diff
@@ -137,6 +144,16 @@ String makePreparedQueryString(TableMetadata table, EntityMapper<?> mapper) {
         }
         throw new AssertionError();
     }
+    
+    private void addSaveOptions(Insert insert, SaveOptions so){
+        if (so.getTtlValue() != -1 && so.getTimestampValue() != -1) {
+            insert.using(ttl(bindMarker("ttlvalue"))).and(timestamp(bindMarker("tsvalue")));
```

Comment:
Named bind markers won't work with protocol v1. As far as I can tell, the mapper has supported it this far.